README.md

```diff
@@ -0,0 +1,20 @@
+# Building
+
+## Build Pre-requisites
+
+The build instructions assume you have NimSkull, latest preferred unless
+otherwise noted, installed and on the path.
+
+## Build Instructions
+
+This command will get everything built
+(N.B.: Windows users replace `nim` with `nim.exe` in the command below)
```

Comment:
```suggestion
To get everything built, run the following from the command line
```
It's a little bit more clear and accessible this way, I think. Also, the `.exe` file extension for the executable is optional on Windoes - at least with the default cmd and PowerShell.

Reply:
Oh neat, I learned something new